### PR TITLE
Cleanup knife-windows configuration options

### PR DESCRIPTION
**Pull request by curiositycasualty**
- :atribute -> :winrm_attribute
- fallback to '(winrm_)<attr>' in #locate_config_value

```
 rspec

<snip>

Finished in 0.77848 seconds (files took 2.59 seconds to load)
118 examples, 0 failures

 knife --version
Chef: 12.13.37
```

/cc @adamedx

### CODE REVIEW

File: lib/chef/knife/bootstrap_windows_ssh.rb [hunk 1]
```diff
@@ -106,7 +106,7 @@ def run_command(command = '')
         ssh.config[:identity_file] = config[:identity_file]
         ssh.config[:ssh_identity_file] = config[:ssh_identity_file] || config[:identity_file]
         ssh.config[:forward_agent] = config[:forward_agent]
-        ssh.config[:manual] = true
```
Comment (mwrock) [inline lib/chef/knife/bootstrap_windows_ssh.rb]: needs to align with https://github.com/chef/chef/blob/master/lib/chef/knife/ssh.rb#L56

File: lib/chef/knife/winrm_shared_options.rb [hunk 2]
```diff
@@ -27,16 +27,16 @@ module WinrmSharedOptions
       # Shared command line options for knife winrm and knife wsman test
       def self.included(includer)
         includer.class_eval do
-          option :manual,
+          option :manual_list,
```
Comment (mwrock) [inline lib/chef/knife/winrm_shared_options.rb]: needs to align with https://github.com/chef/chef/blob/master/lib/chef/knife/ssh.rb#L56

File: lib/chef/knife/winrm_shared_options.rb [hunk 3]
```diff
@@ -27,16 +27,16 @@ module WinrmSharedOptions
...
             :short => "-m",
             :long => "--manual-list",
             :boolean => true,
             :description => "QUERY is a space separated list of servers",
             :default => false
 
-          option :attribute,
-            :short => "-a ATTR",
-            :long => "--attribute ATTR",
+          option :winrm_attribute,
```
Comment (mwrock) [inline lib/chef/knife/winrm_shared_options.rb]: needs to align with https://github.com/chef/chef/blob/master/lib/chef/knife/ssh.rb#L50

Comment (curiositycasualty) [inline lib/chef/knife/winrm_shared_options.rb]: Ah but you don't, especially in light of my changes to lib/chef/knife/knife_windows_base.rb #locate_config_value and the fact that ssh.rb's `:attribute` resolves to `:ssh_attribute`. See: #290 

Namespacing these attributes (both here and in core ssh) allows the code to be more explicit and resolves the stated conflict with `knife node show`.

File: lib/chef/knife/winrm_shared_options.rb (hunk 2 above)
Comment (curiositycasualty) [inline lib/chef/knife/winrm_shared_options.rb]: I can see why this one you'd want to align tho-- lemme fix that.

File: lib/chef/knife/bootstrap_windows_ssh.rb (hunk 1 above)
Comment (curiositycasualty) [inline lib/chef/knife/bootstrap_windows_ssh.rb]: fixed

File: lib/chef/knife/winrm_shared_options.rb (hunk 3 above)
Comment (mwrock) [inline lib/chef/knife/winrm_shared_options.rb]: I'm just saying that the `:option`, short and long arg names on the cli need to be the same in `knife winrm` and `knife ssh`.

**Pull request closed without merging**